docs/man/docker-images.1.md

```diff
@@ -34,7 +34,9 @@ versions.
    Show all images (by default filter out the intermediate image layers). The default is *false*.
 
 **-f**, **--filter**=[]
-   Provide filter values (i.e., 'dangling=true')
+   Provide filter values. Valid filters:
```

Comment:
You can use Filter a  verb here.  Also, it isn't clear that you were trying to describe what dangling=true represents.

--->

Filter output. The filter `dangling=true` finds unlabeled images without children (`label=` or `label==`).